joomla/joomla-cms
# [4.0] Form fields prepared statements, Part 1

Pull Request for Issue # .

### Summary of Changes

Adds prepared statements and cleans up queries in some form fields.

### Testing Instructions

Install testing sample data. Go to backend.

1) Go to Tags list. Inspect `Select Tag Type` field.
2) Go to articles list. Inspect `Select Author` field.
3) Edit a module. Inspect `Module Style` field.
4) Go to Articles configuration. Inspect `Choose a Layout` field.
5) Go to User manager, click on Permissions icon. Inspect `Select Component` field.
6) Create Tagged Items menu item. In the form inspect `Select Content Type` field.
7) Go to Template Manager. Edit a template file. Check that CodeMirror editor appears.

### Expected result

Works like before.

### Documentation Changes Required

No.
